welcomescreen.py

```diff
@@ -20,92 +20,92 @@
 # Contact information:
 # Utkarsh Tiwari    iamutkarshtiwari@gmail.com
 
+import os
+import sys
+
+import pygame
 import gi
 gi.require_version('Gtk', '3.0')
 from gi.repository import Gtk
-import pygame
-import sys
-from gettext import gettext as _
 
+FPS = 60
+BLACK = (0, 0, 0)
 
-class welcomescreen:
+WELCOMESCREEN_PATH = os.path.join("assets", "welcomescreen.png")
+PLAY_BUTTON_PATH = os.path.join("assets/play.png")
+RULESCREEN_PATH = os.path.join("assets/rules.png")
```

Comment:
Make these consistent with the other uses.

Reply:
Fixed